lldb/test/API/lang/swift/async/formatters/task/TestSwiftTaskSyntheticProvider.py

```diff
@@ -21,7 +21,7 @@ def test_top_level_task(self):
             patterns=[
                 textwrap.dedent(
                     r"""
-                    \(Task<\(\), Error>\) task = id:(\d+) flags:(?:running\|)?enqueued\|future \{
+                    \(Task<\(\), Error>\) task = id:(\d+) flags:(?:running\|)?(?:enqueued\|)?future \{
```

Comment:
Actually, I'm still confused by this regex.
Why is the `|` part of the group?
What if enqueued is the last attribute?

Reply:
1. the flags are in a fixed order, `enqueued` will be after `running` (by design)
2. the flags are `|` separated, and if `enqueued` is present, then it will be followed by the `|`

Reply:
I plan to revisit these tests. This change (https://github.com/llvm/llvm-project/pull/131475) is one of the pieces I'm working on in spare time be able to structure these tests with more easy to read regex.

Reply:
ah.. because `future` is always there. Makes sense.